src/Explorer/Menus/CommandBar/CommandBarComponentButtonFactory.tsx

```diff
@@ -309,6 +315,7 @@ function createNewDatabase(container: Explorer): CommandButtonComponentProps {
     commandButtonLabel: label,
     ariaLabel: label,
     hasPopup: true,
+    disabled: useSelectedNode.getState().isQueryCopilotCollectionSelected() ? true : false,
```

Comment:
We can just set disabled directly to `useSelectedNode.getState().isQueryCopilotCollectionSelected()` here right?
`disabled: useSelectedNode.getState().isQueryCopilotCollectionSelected()`